### DISCUSSION TITLE
Ability to customize the Shift+enter menu.

### DISCUSSION THREAD

**Discussion by monandszy** [Flow-Launcher/Flow.Launcher#2518] [Ideas]
Currently, the first option that appears in is to run an app as a different user - not very useful. An ability to customize the order of this menu or a change in order so that it is set to open in explorer or run as administrator as a first result will save me a ton of arrow clicks a day.

**Comment by jjw24** [comment]
Actually the more times you select the result, the higher it will show up, this should be also true for context menu results.

**Reply by monandszy** [reply]
That doesn't seem to be the case for me, I've been using it for some time now, never using the run as different user feature. I'm on the latest version.

**Comment by z1nc0r3** [comment]
This may be out of the question, but you can always run an application as **Administrator** by using `Ctrl + Shift + Enter`.

**Reply by monandszy** [reply]
That's actually perfect! You can also open in folder with Ctrl + F + Enter. Thanks for opening my eyes to this.

**Reply by monandszy** [reply]
Update: you can also do it with only Ctrl + Enter
